Project.toml

```
@@ -1,6 +1,6 @@
 name = "StructArrays"
 uuid = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
-version = "0.6.16"
+version = "0.6.18"
```

```suggestion
version = "0.7.0"
```

There are already some mildly breaking changes on master, so best be safe and bump the minor number as well (see https://github.com/JuliaArrays/StructArrays.jl/issues/291#issuecomment-1864729111)